src/templates/Brochure/BrochureTemplate.tsx

```diff
@@ -343,24 +345,28 @@ const Page4: React.FC<{
                   />
                 </li>
                 <br />
-              </>
+              </React.Fragment>
             );
           }
           return (
-            <>
+            <React.Fragment key={i}>
               <li>
                 <div className="inline-grid">
                   <RedactableValue
                     editable={editable}
                     value={item.description}
-                    onRedactionRequested={() => handleObfuscation(`page4.contents[1].listItems[${i}].description`)}
+                    onRedactionRequested={() =>
+                      handleObfuscation(
+                        `${isV4 ? "credentialSubject." : ""}page4.contents[1].listItems[${i}].description`
+                      )
```

Comment:
obfuscation needs to have the specific string say v2 `foo.bar.x` or v3 / v4 `credentialSubject.foo.bar.x`, i dont have a better solution at this point of time